config/jobs/kubernetes/sig-cloud-provider/gcp/gce-conformance.yaml

```diff
@@ -37,6 +37,178 @@ periodics:
         requests:
           cpu: 1
           memory: 3Gi
+      env:
+      # cncf/apisnoop needs to consume audit logs from the cluster under test in order to
+      # effectively report on conformance test coverage of api operations. The default audit
+      # policy excludes events from being logged due to performance reasons. For this
+      # job we will accept the performance hit and allow events to be logged so that we can
+      # measure coverage of api operations that involve events.
+      #
+      # We do this by setting ENABLE_APISERVER_ADVANCED_AUDIT which tells
+      # cluster/gce/gci/configure-kubeapiserver.sh to override the default audit policy with the
+      # contents of the ADVANCED_AUDIT_POLICY env var.  We've copy-pasted the default
+      # audit policy as output by https://github.com/kubernetes/kubernetes/blob/b1e3a2ac7a80d49ef6153457eaa35dac4fc98444/cluster/gce/gci/configure-helper.sh#L1077-L1235
+      # and then removed the block that says "don't log events"
+      #
+      # NOTE: if the default audit policy changes, this will need to be updated accordingly
+      # TODO(link-to-a-github-issue): ideally we could do this via a patch instead
```

Comment:
I meant to imply you should make a github issue and link to it here

Reply:
Sorry I missed that: https://github.com/kubernetes/test-infra/issues/19613 created and added to the comment.